client/src/components/ProjectWizard/TdmCalculationWizard.js

```diff
@@ -321,6 +363,14 @@ const TdmCalculationWizard = props => {
               ) : null}
             </div>
           ) : null}
+          {page === 5 ? (
+            <div className={classes.lastSavedContainer}>
+              <span className={classes.lastSaved}>
```

Comment:
If a user is creating a project for the first time, there shouldn't be a need to have a "Last Modified" date displayed. So you could have done something like this 
```{dateModified &&  <span className={classes.lastSaved}> ....</span>}``` 
to make it conditionally render the span (assuming that the default value of dateModified is an empty string which is falsey).

Reply:
Nice, thanks Claire!